### ISSUE TITLE
Sound issues

### ISSUE THREAD

**Issue by ZARk-be** [peteward44/WebNES#5]
Hello

I love your emulator (love the post-processing effects ! )

But i keep getting jitters in the sound.
At first i thought it was because i was running a PAL rom, but even with an NTSC rom i get strange jitters.
I tryed with another machine and same issue. Is this due to newer JS engine in Chrome ?

**Comment by peteward44** [comment]
Yes the audio is FPS-dependent, so if your frame rate isn't consistent it will cause the audio to jitter. I wasn't able to recreate this issue on my PC but i'll keep the issue open to see if anyone else has this problem too

**Comment by ZARk-be** [comment]
all machines i test with windows10+chrome have the issue.
testing with different NTSC and PAL roms.
what system do you test this on  ?

**Comment by ZARk-be** [comment]
the only js nes emulator i've found that doesn't have FPS/sound jitters is this one :
https://github.com/bfirsh/jsnes-web/blob/master/src/FrameTimer.js
